src/content/staking/solo/index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Solo stake your ETH
+description: An overview of how to get started solo staking your ETH
+lang: en
+template: staking
+emoji: ":money_with_wings:"
+sidebar: true
+image: ../../../assets/staking/leslie-solo.png
+alt: Leslie the rhino on her own computer chip.
+sidebarDepth: 2
+summaryPoints:
+  - Receive maximum rewards directly from the protocol (including unburnt fees after The Merge) for keeping your validator properly functioning and online
+  - Run home hardware and personally add to the security and decentralization of the Ethereum network
+  - Remove trust, and never give up control of the keys to your funds
+---
+
+## What is solo staking? {#what-is-solo-staking}
+
+Solo staking is the act of running an Ethereum node connected to the internet and depositing 32 ETH to activate a [validator](#faq), giving you the ability to participate directly in network consensus.
+
+An Ethereum node consists of both an execution layer (EL) client, as well as a consensus layer (CL) client. These clients are software that work together, along with a valid set of signing keys, to verify transactions and blocks, attest to the correct head of the chain, aggregate attestations, and propose blocks.
```

Comment:
```suggestion
An Ethereum node consists of both an execution layer client, as well as a consensus layer (CL) client. These clients are software that work together, along with a valid set of signing keys, to verify transactions and blocks, attest to the correct head of the chain, aggregate attestations, and propose blocks.
```

Should just drop the (EL) & (CL)? I don't think we refer to them using those acronyms anywhere & it's probably not ideal for translations anyway.

Reply:
Yeah, perhaps this can be a separate endeavor to try and clean up some of these terms... EL and CL is so much easier to work with, but I agree we need to make sure users are properly introduced to this somewhere before we potentially use it....
Guess that's what I was trying to do here, but never really utilized it elsewhere. Can remove for now and reconsider separately if we want.